Octokit/Models/Request/SearchIssuesRequest.cs

```diff
@@ -337,4 +336,74 @@ public enum IssueTypeQualifier
         [Parameter(Value = "issue")]
         Issue
     }
+
+    public class RepositoryCollection : IEnumerable<string>
```

Comment:
Since the repository list will always be a finite collection, should this be an `IReadOnlyCollection<string>`? And could it not simply inherit a concrete class at that point?